test/underscore.html

```diff
@@ -300,7 +300,7 @@
           '_.escape & unescape': [
             '` is escaped',
             '` can be unescaped',
-            'can escape multiple occurances of `',
+            'can escape multiple occurrences of `',
```

Comment:
This is actually how the test is named in the Underscore test file so we'll need to keep that
but no worries, I got it.